src/components/DownloadMailingListButton.tsx

```diff
@@ -21,10 +21,12 @@ export function DownloadMailingListButton (props: DownloadButtonProps) {
         let owner: OwnerType = ownerDetails[1]
 
         if (typeof contact === 'undefined' || null) {
```

Comment:
|| null doesn't do anything. You can do `if (!contact)` and it will handle both null and undefined